picking_good_slis.md

```diff
@@ -0,0 +1,86 @@
+# Picking Good SLIs
+
+## SRE's Philosophy on Monitoring
+
+A core goal of Site Reliability Engineering (SRE) is to ensure the availability of a service. "Availability" here means much more than simply "is my application running." Effectively monitoring a modern cloud application requires insight into the application's response time, error rate, traffic level, and more.
+
+Furthermore, the scope of monitoring should not be limited solely for use in deciding when to page an operations team. Metrics should be closely tied to business objectives, informing decisions on what features to prioritize and when to focus on adding new features versus improving reliability.
+
+To make this vision a reality, SRE teams rely on two key concepts:
```

Comment:
Is saying "SRE teams" here unnecessarily narrow in scope? Is it reasonable to expand the scope to "service teams" (or similar) with the goal of looping in the dev team(s) reasponsible for the service?

My thinking is that in the forthcoming SLO RACI chart, the personas responsible for executing the SLO roadmap are both Dev and SRE ICs. The execution of the roadmap may include instrumenting the service in question, and I would think that would best fall to the development team of that service.

Reply:
yeah, s/SRE/service